src/planning/plan/tests/plan-consumer-test.ts

```diff
@@ -8,27 +8,23 @@
  * http://polymer.github.io/PATENTS.txt
  */
 import {assert} from '../../../platform/chai-web.js';
-import {Manifest} from '../../../runtime/manifest.js';
 import {Modality} from '../../../runtime/arcs-types/modality.js';
 import {Relevance} from '../../../runtime/relevance.js';
 import {Runtime} from '../../../runtime/runtime.js';
 import {storageKeyPrefixForTest} from '../../../runtime/testing/handle-for-test.js';
-import {Loader} from '../../../platform/loader.js';
 import {PlanConsumer} from '../../plan/plan-consumer.js';
 import {Planificator} from '../../plan/planificator.js';
 import {PlanningResult} from '../../plan/planning-result.js';
 import {Suggestion} from '../../plan/suggestion.js';
 import {SuggestFilter} from '../../plan/suggest-filter.js';
-import {RamDiskStorageDriverProvider} from '../../../runtime/storage/drivers/ramdisk.js';
-import {TestVolatileMemoryProvider} from '../../../runtime/testing/test-volatile-memory-provider.js';
-import {DriverFactory} from '../../../runtime/storage/drivers/driver-factory.js';
 import {Arc} from '../../../runtime/arc.js';
+import {Manifest} from '../../../runtime/manifest.js';
 import {ActiveSingletonEntityStore} from '../../../runtime/storage/storage.js';
 
-async function createPlanConsumer(arc: Arc) {
+async function createPlanConsumer(arc: Arc, context: Manifest) {
   const store: ActiveSingletonEntityStore = await Planificator['_initSuggestStore'](arc);
   assert.isNotNull(store);
-  const result = new PlanningResult({context: arc.context, loader: arc.loader, storageManager: arc.storageManager}, store);
+  const result = new PlanningResult({context, loader: arc.loader, storageManager: arc.storageManager}, store);
```

Comment:
what does it mean for the PlanningResult have a `context` other than the `context` of the Arc it's planned for?